packages/main/src/ComboBoxPopover.hbs

```diff
@@ -70,6 +70,7 @@
 		{{#each _filteredItems}}
 			<ui5-li
 				type="Active"
+				.info={{this.additionalText}}
```

Comment:
you can set it as attribute - "info", (no need to use the property syntax ".info")